ISM43362/ATParser/ATParser.cpp

```diff
@@ -353,7 +353,7 @@ bool ATParser::vrecv(const char *response, va_list args)
 
             // Check for match
             int count = -1;
-            if (whole_line_wanted && c != '\n') {
+            if (whole_line_wanted && c != '\n' &&  c != '\r') {
```

Comment:
add a comment in commit message about why this modification is needed.
i.e. Provide the exact format of response from WIFI module, ending with \r while all other commands are ending with \n

Reply:
Done: inline comment